google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, global_configs, scanner_configs, service_config,
         self.model_name = model_name
         self.snapshot_timestamp = snapshot_timestamp
 
+        LOGGER.debug(str(self.scanner_configs))
```

Comment:
Nit: You can do LOGGER.debug('Scanner configs: %s', self.scanner_configs), rather than casting to str directly.